source/B2BApi.AppTests/Fixtures/Extensions/HttpRequestExtensions.cs

```diff
@@ -88,6 +91,35 @@ public static Task<HttpRequestMessage> CreateDequeueHttpRequestAsync(
             $"api/dequeue/{messageId}");
     }
 
+    public static async Task<HttpRequestMessage> CreateEnqueueCalculatedMeasurementsHttpV1RequestAsync(
+        this B2BApiAppFixture fixture,
+        EnqueueCalculatedMeasurementsHttpV1 enqueueCalculatedMeasurementsHttpV1,
+        string calculationTypeName)
+    {
+        HttpRequestMessage? request = null;
+        try
+        {
+            request = await CreateHttpRequestAsync(
+                fixture,
+                new Actor(
+                    ActorNumber.Create(enqueueCalculatedMeasurementsHttpV1.Data.First().Receivers.First().ActorNumber),
+                    ActorRole.FromName(enqueueCalculatedMeasurementsHttpV1.Data.First().Receivers.First().ActorRole.Name)),
+                HttpMethod.Post,
+                $"api/enqueue/brs021/calculations/{calculationTypeName}",
```

Comment:
```suggestion
                $"api/enqueue/enqueueCalculatedMeasurementsHttpV1.RouteName",
```
https://github.com/Energinet-DataHub/opengeh-process-manager/pull/386/files#diff-5c320cdb59097fb4c1a3dfe1cb057032b634025449634b9613377a09c290bd34R42
Paths should be alignend